src/t8_element_cxx.hxx

```diff
@@ -131,7 +131,7 @@ public:
    * \return          The number of siblings of \a element.
    * Note that this number is >= 1, since we count the element itself as a sibling.
    */
-  virtual int         t8_element_num_siblings (const t8_element_t *
+  virtual int         t8_element_max_num_siblings (const t8_element_t *
                                                elem) const = 0;
 
```

Comment:
We should get rid of this function if we can and use the new num_siblings function from develop.
See comments in adapt

Reply:
Removed it